core/services/scan.go

```diff
@@ -0,0 +1,105 @@
+package services
+
+import (
+	"context"
+	"errors"
+
+	"github.com/kubescape/go-logger"
+	"github.com/kubescape/go-logger/helpers"
+	"github.com/kubescape/kubevuln/core/domain"
+	"github.com/kubescape/kubevuln/core/ports"
+	"go.opentelemetry.io/otel"
+)
+
+type ScanService struct {
+	sbomCreator    ports.SBOMCreator
+	sbomRepository ports.SBOMRepository
+	cveScanner     ports.CVEScanner
+	cveRepository  ports.CVERepository
+	armoPlatform   ports.Platform
```

Comment:
issue: CNCF requires us to be vendor-neutral, so we should be agnostic of specific backends

We even had to remove all mentions of ARMO in Kubevuln, so adding them back is a no-no :)

Reply:
`saasPlatform` then?